src/ui/public/vislib/visualizations/_map.js

```diff
@@ -1,24 +1,28 @@
 import _ from 'lodash';
 import $ from 'jquery';
 import L from 'leaflet';
+import marked from 'marked';
+marked.setOptions({
+  gfm: true, // Github-flavored markdown
+  sanitize: true // Sanitize HTML tags
+});
+
 import VislibVisualizationsMarkerTypesScaledCirclesProvider from 'ui/vislib/visualizations/marker_types/scaled_circles';
 import VislibVisualizationsMarkerTypesShadedCirclesProvider from 'ui/vislib/visualizations/marker_types/shaded_circles';
 import VislibVisualizationsMarkerTypesGeohashGridProvider from 'ui/vislib/visualizations/marker_types/geohash_grid';
 import VislibVisualizationsMarkerTypesHeatmapProvider from 'ui/vislib/visualizations/marker_types/heatmap';
-export default function MapFactory(Private) {
+export default function MapFactory(Private, tilemap) {
 
   let defaultMapZoom = 2;
   let defaultMapCenter = [15, 5];
   let defaultMarkerType = 'Scaled Circle Markers';
 
+  let tilemapOptions = tilemap.options;
+  let attribution = marked(tilemapOptions.attribution);
```

Comment:
@spalger no, because the sanitize option is enabled.

When I used the string 
```
'Map data: &copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>, <a href="http://viewfinderpanoramas.org">SRTM</a> | Map style: &copy; <a href="https://opentopomap.org">OpenTopoMap</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a>)'
```

The output on the page was the literal string, html tags and all.